src/main/java/edu/uc/inventorymanager/dto/User.java

```diff
@@ -3,6 +3,6 @@
 import lombok.Data;
 
 public @Data class User {
-    private int id;
-    private String name;
+    private int userId;
+    private String userName;
```

Comment:
Cleaning variable names in accordance to Baeldung - Clean Code

See: dto.Item comments.

Reply:
Ditto above.  I'd leave this as it were.